eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/activities/MicroFragmentHostActivity.java

```diff
@@ -46,6 +50,8 @@
     protected void onCreate(Bundle savedInstanceState)
     {
         super.onCreate(savedInstanceState);
+
+        // TODO No need to set layout, android.R.id.content can be used for the fragment containerId
```

Comment:
I think it's safe to do, they say it here as well:
https://stackoverflow.com/questions/24712227/android-r-id-content-as-container-for-fragment

So I changed it now, for `LocationPickerActivity` as well, and removed the layout.